### PR TITLE
Add cors dependency to allow localhost development

### PR DESCRIPTION
**Pull request by nashirj**

### CODE REVIEW

File: server/App.ts [hunk 1]
```diff
@@ -23,6 +23,14 @@ app.use(Logger.consoleLog);
 
 app.use(express.static(path.resolve(__dirname, '../frontend/html/public')));
 
+const cors = require("cors");
+
+var corsOptions = {
+  origin: "http://localhost:8081"
+};
+
+app.use(cors(corsOptions));
```
Comment (nashirj) [inline server/App.ts:32]: this needs to be wrapped in something like `if process.env.NODE_ENV == 'dev'` or something like that

Comment (nashirj) [reply server/App.ts:32]: Actually, this is needed for localhost frontend development to work when interacting with the deployed server as well.

**Review by mplekunov — APPROVED** [review]
lgtm :+1:

**Comment by mplekunov** [comment]
ib4 server went down :rofl:

**Pull request merged**